README.md

```diff
@@ -18,22 +18,15 @@ Some of the thinking behind the router is documented in [this 2013 blog post][po
 [router-api]: https://github.com/alphagov/router-api
 [post]: https://gdstechnology.blog.gov.uk/2013/12/05/building-a-new-router-for-gov-uk/
 
-## Local Setup & Build
+## Technical documentation
 
-The router needs to be built from within a Go [workspace][workspace] for it to
-correctly identify the vendored dependencies.  Assuming you already have Go and
-a `$GOPATH` set up, you can simply run `go get github.com/alphagov/router` to
-pull the source into your workspace.
+Recommended reading: [How to Write Go Code](https://golang.org/doc/code.html)
 
-Once you have the code set up correctly, building and running the application
-is simple:
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
-    make
-    ./router -h
```

Comment:
Nice. Might be worth noting in the commit that we don't lose any information by deleting this section - [govuk-docker already runs `./router` via govuk-docker-up](https://github.com/alphagov/govuk-docker/blob/51e040b25d738f8e832b39624ece5fff697c3b96/projects/router/docker-compose.yml#L40). (I suspected that was the case but wanted to double check and make sure, given this is a unique app).

Reply:
Sure, I've done that, and also suggested we can reinstate them if non-Docker dev is unclear/desirable in future. However, as I discovered when doing this PR, it's important we push people to use GOV.UK Docker, as otherwise it can get broken unintentionally, which I've explained in this issue: https://github.com/alphagov/router/issues/210

Anyway, I'll merge this now, and you're welcome to deploy 😄.